cmd/mms/commands.go

```diff
@@ -138,3 +146,39 @@ func productReceiver(event *mms.ProductEvent) error {
 	fmt.Println(event)
 	return nil
 }
+
+func createExecutableCallback(filepath string, filter bool) func(event *mms.ProductEvent) error {
+	_, err := exec.LookPath(filepath)
+	fmt.Println(filepath)
+	if err != nil {
+		panic(fmt.Errorf("command executable not found, %s", err.Error()))
+
+	}
+
+	return func(event *mms.ProductEvent) error {
+		var productslug string
+
+		if filter {
+			productslug = event.Product
+		} else {
+			productslug = ""
+		}
+		command := exec.Command(filepath, productslug)
```

Comment:
For the processes not to hang forever, we should add a timeout, this we can do in another iteration though:
https://github.com/metno/go-mms/issues/116

Reply:
Good point, defered to #116